core/modules/cms-prismic/types/CmsBlockState.ts

```diff
@@ -0,0 +1,4 @@
+export default interface CmsBlockState {
+  type: any[],
```

Comment:
don't use type `any` if it's not really any possible type. You can use union types if sth can have more than one possible type